CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - UpgradeSteps should be filtered based on project components [#255](https://github.com/dotnet/upgrade-assistant/issues/255)
 - Do not add _ViewImports.cshtml to VB projects [#378](https://github.com/dotnet/upgrade-assistant/issues/378)
+- Now mocking package manager versions for integration tests [#416](https://github.com/dotnet/upgrade-assistant/issues/416)
```

Comment:
Is this user facing?

Reply:
Agreed, this does not fall into user facing category and probably does not need to be called out in the changelog.